Sources/PackageRegistryTool/PackageRegistryTool+Publish.swift

```diff
@@ -153,18 +153,12 @@ extension SwiftPackageRegistryTool {
                 )
             }
 
-            if publishConfiguration.signing.privateKeyPath == nil {
+            if publishConfiguration.signing.privateKeyPath != nil {
                 guard !publishConfiguration.signing.certificateChainPaths.isEmpty else {
                     throw StringError(
                         "Both 'privateKeyPath' and 'certificateChainPaths' are required when one of them is set."
                     )
                 }
-            } else {
-                guard publishConfiguration.signing.certificateChainPaths.isEmpty else {
-                    throw StringError(
-                        "Both 'privateKeyPath' and 'certificateChainPaths' are required when one of them is set."
-                    )
-                }
```

Comment:
iiuc correctly they need to be both set or both not-set? if so, a fix could be:

```
if publishConfiguration.signing.privateKeyPath != nil {
                guard !publishConfiguration.signing.certificateChainPaths.isEmpty else {
                    throw StringError(
                        "Both 'privateKeyPath' and 'certificateChainPaths' are required when one of them is set."
                    )
                }
            } else {
                guard publishConfiguration.signing.certificateChainPaths.isEmpty else {
                    throw StringError(
                        "Both 'privateKeyPath' and 'certificateChainPaths' are required when one of them is set."
                    )
                }
            }
```

Reply:
The else guard would fail if omit `privateKeyPath` and `certificateChainPaths` in the call, right? Since `certificateChainPaths` defaults to `[]` and I don't think we can change the default to `nil`.